lib/Target/CBackend/CMakeLists.txt

```diff
@@ -17,4 +17,5 @@ add_llvm_target(CBackendCodeGen
   CBackend.cpp
   CTargetMachine.cpp
   TopologicalSorter.cpp
+  TargetInfo/CBackendTargetInfo.cpp
```

Comment:
this fix is needed because atleast in llvm 13+14 which still detect llvm-cbe as a possible backend if compiled as `llvm-project\llvm\projects\llvm-cbe`